src/main/java/com/halo/core_bridge/api/jobposting/model/dto/JobPostingDto.java

```diff
@@ -0,0 +1,104 @@
+package com.halo.core_bridge.api.jobposting.model.dto;
+
+import com.halo.core_bridge.api.jobposting.model.entity.JobPosting;
+import com.halo.core_bridge.api.jobposting.model.entity.JobPostingSkill;
+import com.halo.core_bridge.api.organization.model.entity.Department;
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class JobPostingDto {
+
+    @Getter
+    @Setter
+    @NoArgsConstructor
+    @AllArgsConstructor
+    @Builder
+    public static class CreateRequest {
+        private String title;
+        private String description;
+        private String employmentType;
+        private String careerType;
+        private int minExperience;
+        private int maxExperience;
+        private Long departmentId;
+        private LocalDateTime applyStartDate;
+        private LocalDateTime applyEndDate;
+        private LocalDateTime hireEndDate;
+        private List<String> skills;
+
+        public JobPosting toEntity(Department department) {
+            JobPosting posting = JobPosting.builder()
+                    .title(this.title)
+                    .description(this.description)
+                    .employmentType(this.employmentType)
+                    .careerType(this.careerType)
+                    .minExperience(this.minExperience)
+                    .maxExperience(this.maxExperience)
+                    .applyStartDate(this.applyStartDate)
+                    .applyEndDate(this.applyEndDate)
+                    .hireEndDate(this.hireEndDate)
+                    .department(department)
+                    .build();
+
+            if (skills != null) {
+                this.skills.forEach(skillName ->
+                        posting.addSkill(JobPostingSkill.builder().name(skillName).build()));
+            }
+            return posting;
+        }
+
+    }
+
+    // 공고 목록 응답
+    @Getter
+    @NoArgsConstructor @AllArgsConstructor @Builder
```

Comment:
현재 클래스에 Lombok 어노테이션이
@Getter, @NoArgsConstructor, @AllArgsConstructor, @Builder
이렇게 한 줄에 모여 있는데,
각 클래스마다 어노테이션 일관성을 가져가면서 각 어노테이션의 역할을 명확히 보기 위해 한 줄씩 나누어 작성하면 좋을 것 같아요!
```suggestion
    @Getter
    @NoArgsConstructor 
    @AllArgsConstructor 
    @Builder
```